samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -17,6 +17,7 @@ The sample supports the following development kits:
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
    :rows: nrf52833dk_nrf52833
```

Comment:
`:rows: nrf52833dk_nrf52833, nrf52833dk_nrf52820`

Reply:
Corrected, thank you.